app/controllers/prior_authority/applications_controller.rb

```diff
@@ -1,12 +1,11 @@
 module PriorAuthority
   class ApplicationsController < ApplicationController
     before_action :set_default_table_sort_options
-    before_action :load_drafts, only: %i[index draft]
-    before_action :load_reviewed, only: %i[index reviewed]
-    before_action :load_submitted, only: %i[index submitted]
     layout 'prior_authority'
 
     def index
+      @pagy, @model = order_and_paginate(PriorAuthorityApplication.for(current_provider).reviewed)
```

Comment:
This may be overthinking, but I'm wondering if we can DRY this up as follows. If we adjust order_and_paginate to look like this:

```ruby
def  order_and_paginate
  query = yield PriorAuthorityApplication.for(current_provider)
  # ... same as before from here on
end

Then in our actions we can do: 

```ruby
@pagy, @model = order_and_paginate(&:reviewed)
```

So in the action we define a block that tells `order_and_paginate` what method to use to filter applications on.